server/sonar-server/src/main/java/org/sonar/server/test/index/TestIndex.java

```diff
@@ -114,7 +115,7 @@ public TestDoc getByTestUuid(String testUuid) {
       .setTypes(TestIndexDefinition.TYPE)
       .setSize(searchOptions.getLimit())
       .setFrom(searchOptions.getOffset())
-      .setQuery(QueryBuilders.filteredQuery(QueryBuilders.matchAllQuery(), FilterBuilders.termFilter(FIELD_TEST_UUID, testUuid)));
+      .setQuery(QueryBuilders.filteredQuery(QueryBuilders.matchAllQuery(), termQuery(FIELD_TEST_UUID, testUuid)));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this use of "filteredQuery"; it is deprecated. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)